src/accordion/accordion.ts

```diff
@@ -94,8 +125,10 @@ export interface NgbPanelChangeEvent {
   <div class="card">
     <template ngFor let-panel [ngForOf]="panels">
       <div [class]="'card-header ' + (panel.type ? 'card-'+panel.type: type ? 'card-'+type : '')" [class.active]="isOpen(panel.id)">
-        <a tabindex="0" href (click)="!!toggle(panel.id)" [class.text-muted]="panel.disabled">
-          {{panel.title}}<template [ngTemplateOutlet]="panel.titleTpl?.templateRef"></template>          
+        <template *ngIf="panel.headerTpl" [ngTemplateOutlet]="panel.headerTpl?.templateRef" [ngOutletContext]="getHeaderContext(panel)">
+        </template>
+        <a *ngIf="!panel.headerTpl" tabindex="0" href (click)="!!toggle(panel.id)" [class.text-muted]="panel.disabled">
```

Comment:
With this change people will have to manage `[class.text-muted]="panel.disabled"` on their side, right? But currently this is not part of the context.

We should explore if we can move `[class.text-muted]="panel.disabled"` higher up or expose it as part of the context.